c/cookies/build_info.json

```diff
@@ -0,0 +1,17 @@
+{
+   "maintainer": "mohit.pawar@ibm.com",
+   "package_name": "cookies",
+   "github_url": "http://github.com/pillarjs/cookies",
+   "version": "0.9.1",
+   "default_branch": "master",
+   "build_script": "cookies_ubi_8.7.sh",
+   "package_dir": "c/cookies",
+   "docker_build": true,
+   "docker_cmd" : "docker build -t ${package_name}:$PACKAGE_VERSION ${dir}",
+   "use_non_root_user": false,
+   "validate_build_script": true,
+   "0.*.*": {
+      "dir": "0.9.1_ubi_9",
+      "build_script": "cookies_ubi_8.7.sh"
```

Comment:
script name should have ubi 9.3 reference.
update same in all places @mohitp333